src/ext/language_tools.js

```diff
@@ -162,10 +162,12 @@ require("../config").defineOptions(Editor.prototype, "editor", {
         value: false
     },
     /**
-     * Enable live autocomplete. If the value is an array, it is assumed to be an array of completers
-     * and will use them instead of the default completers.
+     * Enable live autocompletion
      */
     enableLiveAutocompletion: {
+        /**
+         * @param {boolean} val
```

Comment:
It's still possible to pass an array of completers right?

So instead as suggested in original issue the text should be

```
Enable basic autocomplete. If the value is an array, it is assumed to be an array of completers and will use them unless you already have completers set e.g. from **enableLiveAutocompletion.**
```

Reply:
The api here is rather strange and not very useful, isn't it better to deprecate passing the array. The users can use completers array directly anyway.